code/modules/clothing/masks/_masks.dm

```diff
@@ -36,11 +36,17 @@
 		clothing_flags |= visor_flags
 		flags_inv |= visor_flags_inv
 		flags_cover |= visor_flags_cover
-		to_chat(user, "<span class='notice'>You push \the [src] back into place.</span>")
+		if(src.type == /obj/item/clothing/mask/bandana || src.parent_type == /obj/item/clothing/mask/bandana)
+			to_chat(user, "<span class='notice'>You re-tie the bandana into a mask.</span>")
+		else
+			to_chat(user, "<span class='notice'>You push \the [src] back into place.</span>")

```

Comment:
Remove `\the`

Reply:
The `\the` was there beforehand. I'll remove it and see what happens, and change it here if there's no difference.

Reply:
It seems the `\the` isn't necessary, and serves no purpose. But I'm reverting the change to _masks.dm since the fix I put in place wasn't proper, and it was just a side language fix not really related to the PR.